Dockerfile

```diff
@@ -130,6 +130,17 @@ RUN set -x \
 	&& git clone -b v1.2 https://github.com/russross/blackfriday.git /go/src/github.com/russross/blackfriday \
 	&& go install -v github.com/cpuguy83/go-md2man
 
+# Install registry
+COPY pkg/tarsum /go/src/github.com/docker/docker/pkg/tarsum
```

Comment:
Then I'm not doing my job right.  It's always my fault, so you really
shouldn't trust me. ;)